test/controllers/nonsipp/membercontributions/MemberContributionsCYAControllerSpec.scala

```diff
@@ -16,25 +16,50 @@
 
 package controllers.nonsipp.membercontributions
 
-import services.PsrSubmissionService
+import services.{PsrSubmissionService, SchemeDateService}
 import pages.nonsipp.membercontributions.TotalMemberContributionPage
 import config.Refined.Max300
 import controllers.ControllerBaseSpec
 import play.api.inject.bind
 import views.html.CheckYourAnswersView
 import eu.timepit.refined.refineMV
-import models.{CheckMode, Mode, NormalMode}
+import models._
 import viewmodels.models.SectionCompleted
 import org.mockito.ArgumentMatchers.any
 import play.api.inject.guice.GuiceableModule
 import pages.nonsipp.memberdetails.{MemberDetailsCompletedPage, MemberDetailsPage}
-import org.mockito.Mockito.{reset, times, verify}
+import org.mockito.Mockito._
 
 class MemberContributionsCYAControllerSpec extends ControllerBaseSpec {
```

Comment:
Tests missing, please check "BasicDetailsCheckYourAnswersController in view only mode" as a referece.